src/commands/schedules/modify.js

```diff
@@ -25,14 +25,29 @@ export default (bot, uri) => {
       include: 'Timerange'
     });
 
-    const wh = _.find(workhours, { weekday: moment().weekday() }) || { Timeranges: [] };
-    const timerange = wh.Timeranges[wh.Timeranges.length - 1];
     const date = parseDate(bot, vdate);
 
+    let weekday;
+    if (date && date.range) {
+      if (date.from.isValid()) weekday = date.from;
```

Comment:
Isn't it better to use ternary here?

P.S: It's a good habit to always use if blocks with curly braces.